src/components/card/CardModal.vue

```diff
@@ -0,0 +1,370 @@
+<!--
+  - @copyright Copyright (c) 2018 Julius Härtl <jus@bitgrid.net>
+  -
+  - @author Julius Härtl <jus@bitgrid.net>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<div class="container">
+		<div class="top">
+			<h1 class="top-title">
+				Example task 3
+			</h1>
+			<p class="top-modified">
+				Modified: 2 days go. Created 3 days ago
+			</p>
+		</div>
+		<div class="tabs">
```

Comment:
I think there might have been a bit of confusion on what the purpose of the top buttons is, they are not tabs in specific, but more toggle buttons for the actual input fields. Let me maybe describe the idea there a bit further:

Taking the due date as an example
- By default the input field is hidden
- The user decides to add a due date by clicking the "Due date" button
- Then the input is shown
- If a card has a due date set, we always show the input field

The same would be the case for the others (Members, Projects, Attachments, Tags)